packages/fluent-theme/src/fluent/styles/ComboBox.styles.ts

```diff
@@ -1,13 +1,61 @@
 import { fluentBorderRadius } from './styleConstants';
 import { Depths } from '../FluentDepths';
+import { NeutralColors, SharedColors } from '../FluentColors';
 
 export const ComboBoxStyles = {
   root: {
-    borderRadius: fluentBorderRadius // the bound input box
+    borderRadius: fluentBorderRadius, // the bound input box
+    borderColor: NeutralColors.gray80,
+    paddingLeft: 8
+  },
+  rootHovered: {
+    borderColor: NeutralColors.gray160,
+    selectors: {
+      '.ms-ComboBox-Input': {
+        color: NeutralColors.gray190
+      }
+    }
+  },
+  rootError: {
+    borderColor: SharedColors.red10 // current structure of ComboBox does not allow to change the hover/focus color when has error
   },
   callout: {
-    borderRadius: `0 0 ${fluentBorderRadius} ${fluentBorderRadius}`,
-    overflow: 'hidden',
-    boxShadow: Depths.depth8
+    borderRadius: fluentBorderRadius,
+    border: 'none',
+    boxShadow: Depths.depth8,
+    selectors: {
+      ['.ms-Callout-main']: { borderRadius: fluentBorderRadius }
+    }
+  },
+  header: {
+    padding: '0 8px'
+  },
+  optionsContainer: {
+    selectors: {
+      '.ms-ComboBox-option': {
```

Comment:
why do we have these global selectors? shouldn't we be targeting specific parts?

Reply:
ComboBox has no possibility to style it's options from it's own styles (except if it's a header or divider). Which I believe it's wrong and proper subComponentStyles hooks should be added, but this out of the scope of the fluent effort. Also, hoping that ComboBox and Dropdown merge will eventually become a reality. Plus ContextualMenu should also be taken in consideration when that will happen as according to the designers all the items in the lists they provide should have one common style and in 3 different places with 3 different approaches, causing lots of inconsistencies and pains in maintaining them.